datm/datm_datamode_era5_mod.F90

```diff
@@ -33,7 +33,6 @@ module datm_datamode_era5_mod
   real(r8), pointer :: Sa_shum(:)           => null()
   real(r8), pointer :: Sa_dens(:)           => null()
   real(r8), pointer :: Sa_pbot(:)           => null()
-  real(r8), pointer :: Sa_pslv(:)           => null()
```

Comment:
Please check with Ufuk as to if its okay to remove thes.

Reply:
Variables that are not currently in use are safe to remove.